### PR TITLE
docs: add france and update orange pl

### LINKED ISSUE #175: Working on Pixel 7 pro with NRJ mobile Carrier
Hello,  I got VoWIFI and VOLTE working on NRJ MOBILE, it's a MVNO on Bouygues Telecom network in FRANCE.
I used the Shizuku without PC (Wi-Fi connection required) and in 5 minutes I got Wifi and LTE calls working !
Many thanks for this great patch and app.

### PR DESCRIPTION
**Pull request by fillwithjoy1**
added orange pl to add ims apn. I've just seen too many discussions where wifi calling isn't working unless they added the ims apn.

oh and add compatibility with NRJ mobile carrier (see #175)

close #175

### CODE REVIEW

**Comment by fillwithjoy1** [comment]
okay just wondering, what happens if a carrier has multiple mnc codes?

File: docs/compatibility-chart.en.md [hunk 1]
```diff
@@ -29,10 +29,11 @@ Carrier not being listed on the table does not mean that this patch won't work f
...
 | Europe        | Switzerland (228)       | Salt (03)           | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/125#issuecomment-1546696568)                                                                                                  |
-| Europe        | Poland (260)            | Orange PL (03)      | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/17)                                                                                                                           |
+| Europe        | Poland (260)            | Orange PL (03)      | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/17). If Vo-Wifi isn't working, you may need to add the `ims` APN [Reference](https://github.com/kyujin-cho/pixel-volte-patch/discussions/1#discussioncomment-6261820)                                                                                                                          |
```
Comment (fillwithjoy1) [inline docs/compatibility-chart.en.md]: wait should I change the link to
https://github.com/kyujin-cho/pixel-volte-patch/issues/136#issuecomment-1565598716

Comment (fillwithjoy1) [reply docs/compatibility-chart.en.md]: probably because that was turkcell.

**Review by fillwithjoy1 — COMMENTED** [review]
In #136, there are people who report adding IMS APN works (and they are on orange. maybe im stupid because orange seems to exist upon multiple countries. maybe this entire commit is unnecessary...)

**Comment by kyujin-cho** [comment]
> okay just wondering, what happens if a carrier has multiple mnc codes?

@fillwithjoy1 Just adding every available MNC into single cell will be enough.

**Comment by fillwithjoy1** [comment]
> > okay just wondering, what happens if a carrier has multiple mnc codes?
> 
> @fillwithjoy1 Just adding every available MNC into single cell will be enough.

Okay, I think this pull request is ready to be reviewed now

**Pull request merged**